### DISCUSSION TITLE
📣 제페토 월드 공모전 수상자 발표

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#1065] [1. Announcements]
안녕하세요~ 제페토 에반젤리스트입니다.
월드 개발부터 결과 발표까지 4개월 간의 긴 여정을 함께해 주신 크리에이터 분들께 진심으로 감사 드립니다.
치열한 경쟁을 뚫고 선정된 4개의 월드를 발표합니다! ✨

![style2](https://user-images.githubusercontent.com/100118650/180724627-b209da64-0ff1-45a0-934b-47db14e842a3.png)

### 🏆 대상 : 1000만원 🏆
> 젬토리 : 나의 섬 (GEMTORY : MY ISLAND) by gemtory

![1](https://user-images.githubusercontent.com/100118650/180724998-af6bc82a-74b3-4cb7-a5bb-b6bbff822f2e.png)

### 🏆 최우수상 : 500만원 🏆
> TURN! SPACE OVER by capgreen 

![2](https://user-images.githubusercontent.com/100118650/180727324-aa5a6689-bfca-485c-ae44-47458cdb47f8.png)

### 🏆 우수상 : 300만원 🏆
> 오션랜드 (Ocean Land) by 오션랜드

![3](https://user-images.githubusercontent.com/100118650/180727733-cb67d959-149b-4333-8eac-9cf9b45b3ea2.png)

### 🏆 장려상 : 200만원 🏆
> Catch the train by dongdong55 

![4](https://user-images.githubusercontent.com/100118650/180727770-5125c93d-4769-419f-b517-ea7e2020e374.png)

자세한 내용은 [공지사항](https://page.zepeto.me/5662O3wK875tc5JXnhB0d09?lang=ko)을 확인하여 주세요.
수상자 4팀 모두 축하드립니다!
감사합니다.

**Comment by chlguswls32** [comment]
축하드립니다.
